packages/xrpc-server/tests/frames.test.ts

```diff
@@ -0,0 +1,171 @@
+import * as cborx from 'cbor-x'
+import * as uint8arrays from 'uint8arrays'
+import { DataFrame, ErrorFrame, Frame, FrameType, InfoFrame } from '../src'
+
+describe('Frames', () => {
+  it('creates and parses message frame.', async () => {
+    const messageFrame = new DataFrame({
+      type: 2,
+      body: { a: 'b', c: [1, 2, 3] },
+    })
+
+    expect(messageFrame.header).toEqual({
+      op: FrameType.Data,
+      t: 2,
+    })
+    expect(messageFrame.op).toEqual(FrameType.Data)
+    expect(messageFrame.type).toEqual(2)
+    expect(messageFrame.body).toEqual({ a: 'b', c: [1, 2, 3] })
+
+    const bytes = messageFrame.toBytes()
+    expect([...bytes]).toEqual([
```

Comment:
could do `uint8arrays.equal()` here to avoid the destructuring 

no biggie tho